README.md

```diff
@@ -1,38 +1,70 @@
+<div align="center">
+
 # Local Mirror Detection
-  
-[![Generic badge](https://img.shields.io/badge/Unity-2019.4.31f1-informational.svg)](https://unity3d.com/unity/whats-new/2019.4.31)
-[![Generic badge](https://img.shields.io/badge/SDK-AvatarSDK3-informational.svg)](https://vrchat.com/home/download)
-[![Generic badge](https://img.shields.io/badge/License-MIT-informational.svg)](https://github.com/VRLabs/Local-Mirror-Detection/blob/main/LICENSE)
+
 [![Generic badge](https://img.shields.io/github/downloads/VRLabs/Local-Mirror-Detection/total?label=Downloads)](https://github.com/VRLabs/Local-Mirror-Detection/releases/latest)
+[![Generic badge](https://img.shields.io/badge/License-MIT-informational.svg)](https://github.com/VRLabs/Local-Mirror-Detection/blob/main/LICENSE)
+[![Generic badge](https://img.shields.io/badge/Unity-2019.4.31f1-lightblue.svg)](https://unity3d.com/unity/whats-new/2019.4.31)
+[![Generic badge](https://img.shields.io/badge/SDK-AvatarSDK3-lightblue.svg)](https://vrchat.com/home/download)
+
+[![Generic badge](https://img.shields.io/discord/706913824607043605?color=%237289da&label=DISCORD&logo=Discord&style=for-the-badge)](https://discord.vrlabs.dev/)
+[![Generic badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.vercel.app%2Fapi%3Fusername%3Dvrlabs%26type%3Dpatrons&style=for-the-badge)](https://patreon.vrlabs.dev/)
+
+Sets an "IsMirror" parameter to true on your mirror clone
+
+![VRChat_2023-10-25_20-51-13 163_3840x2160](https://github.com/VRLabs/Local-Mirror-Detection/assets/76777936/2e9e5f00-6b90-40f6-9c8e-44b7be6dd610)
+
+### ⬇️ [Download Latest Version](https://github.com/VRLabs/Local-Mirror-Detection/releases/latest)
+
+<!-- 
+### 📦 [Add to VRChat Creator Companion]() -->
 
-Set an "IsMirror" parameter to animate conditionally with the local mirror.
+</div>
+
+---
 
 ## How it works
 
-The mirror-copy FX layer initializes after the host avatar's FX layer, which can be exploited to set different parameter values.
+* The mirror clone initialized after the local clone by copying the local clone in its current state.
```

Comment:
It should be "the mirror clone ***is*** initialized after..." or "the mirror clone ***initializes*** after..."